backend/accounts.go

```diff
@@ -188,6 +190,63 @@ func (backend *Backend) SupportedCoins(keystore keystore.Keystore) []coinpkg.Cod
 	return availableCoins
 }
 
+// LookupInsuredAccounts checks the insurance status of one or more accounts. If the input
+// account code is not empty, the check is made exclusively on that account, otherwise it is done
+// on all the BTC/TBTC, active, not hidden accounts.
+// In case of updates of the insurance status of any account, the new status is persisted and the
+// accounts are reinitialized.
+// It returns a slice of insured account codes.
+func (backend *Backend) LookupInsuredAccounts(accountCode string) ([]types.Code, error) {
```

Comment:
can this be `accountCode types.Code`? 

I also wonder why it is a string here and immediately converted to the `Code` type:

```
func (backend *Backend) GetAccountFromCode(code string) (accounts.Interface, error) {
	acctCode := accountsTypes.Code(code)
```

Reply:
Good point. Added a new commit to fix that.